zuul.d/projects.yaml

```diff
@@ -168,6 +168,15 @@
       - publish-to-galaxy
       - publish-to-automation-hub
 
+- project:
+    name: ansible-community/ansible.plugin_builder
+    default-branch: main
+    templates:
+      - system-required
+      - ansible-community-ansible-plugin-builder
```

Comment:
Remove this job as of now. We can test it, and then add this job to this template.

Reply:
Made the change. But not sure about the `The nodeset "vmware-vcsa-7.0.3-python36" was not found.` failure.